### PR TITLE
xdg-shell: chase events update

### PR DESCRIPTION
**Pull request by vyivel**
https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4345

### CODE REVIEW

**Comment by vyivel** [comment]
7777 \o/

File: sway/desktop/xdg_shell.c [hunk 1]
```diff
@@ -279,6 +286,19 @@ static void handle_commit(struct wl_listener *listener, void *data) {
 	struct sway_view *view = &xdg_shell_view->view;
 	struct wlr_xdg_surface *xdg_surface = view->wlr_xdg_toplevel->base;
 
+	if (xdg_surface->initial_commit) {
+		if (view->xdg_decoration != NULL) {
+			set_xdg_decoration_mode(view->xdg_decoration);
+		}
+		// XXX: https://github.com/swaywm/sway/issues/2176
+		wlr_xdg_surface_schedule_configure(xdg_surface);
```
Comment (vyivel) [inline sway/desktop/xdg_shell.c:294]: Hm, automatic configuration isn't removed yet so this isn't required.

Comment (vyivel) [reply sway/desktop/xdg_shell.c:294]: It's fine though, Sway will simply be ready for that too.

Comment (vyivel) [reply sway/desktop/xdg_shell.c:294]: https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4396

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**